Xamarin.Forms.Core/Color.cs

```diff
@@ -400,6 +400,24 @@ public static Color FromHsla(double h, double s, double l, double a = 1d)
 		{
 			return new Color(h, s, l, a, Mode.Hsl);
 		}
+
+
+		public object InterpolateTo(object from, object target, double interpolate)
+		{
+			return InterpolateTo((Color)from, (Color)target, interpolate);
+		}
+
+		public Color InterpolateTo(Color from, Color target, double interpolate)
+		{
+
+			var rR = from.R + (target.R - from.R) * interpolate;
+			var gG = from.G + (target.G - from.G) * interpolate;
+			var bB = from.B + (target.B - from.B) * interpolate;
+			var aA = from.A + (target.A - from.A) * interpolate;
+
+			return FromRgba(rR, gG, bB, aA);
```

Comment:
that gives you brown, every time

color animation are better done using HSL

Reply:
I didn't see the problem (get always brown) you are commenting on here Stephane.
Seems to do the color interpolation correctly: https://www.alanzucconi.com/2016/01/06/colour-interpolation/
I did it in another place in a similar way: https://github.com/jsuarezruiz/Xamanimation/blob/master/Xamanimation/Helpers/AnimationHelper.cs#L20